acceptance/tests/subcommands/provision.rb

```diff
@@ -10,8 +10,10 @@ def delete_root_folder_contents
     delete_root_folder_contents
     on(default, 'beaker init')
     result = on(default, 'beaker provision --hosts centos6-64')
+    assert_match(/ERROR/, result.raw_output)
```

Comment:
I'd like to ensure that the error message at least has some reference to the flag that is not allowed. Something like `/ERROR(.+)--hosts/` would work.